### PR TITLE
added header / h1

### PR DESCRIPTION
**Pull request by DErdmann1**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+* {
+    margin: 0px;
```
Comment (vaupunkt) [inline styles.css:2]: you could add 
`box-sizing: border-box;`
to your whole layout

**Review by vaupunkt — APPROVED** [review]
just 1 small comment, the rest looks good 👍

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+header {
+    text-align: center;
+    font-family: serif;
+    font-size: 20px;
+    box-sizing: border-box;
+    border:  5px solid black;
+    height: 80px;
+    padding-top: 10px;
+
```
Comment (josch87) [inline styles.css:15]: You could work with flexbox in order to center your title horizontally.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    font-size: 20px;
+    box-sizing: border-box;
+    border:  5px solid black;
+    height: 80px;
+    padding-top: 10px;
+    
+}
+
+body {
+
```
Comment (josch87) [inline styles.css:19]: If you don't have declarations for this element delete the selector.

File: index.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
     <title>Quiz-App</title>
     <link rel="stylesheet" type="text/css" href="styles.css">
 </head>
-<body>
-    
-</body>
```
Comment (josch87) [inline index.html:12]: Unfortunately you need to have all your content within this element. You shoudln't delete it.

**Review by josch87 — COMMENTED** [review]
I encountered an issue with your code. You will need to have all the content within the body element.

**Pull request merged**